recipes/fmt/all/conanfile.py

```diff
@@ -49,6 +49,8 @@ def config_options(self):
             del self.options.fPIC
         if not self._has_with_os_api_option:
             del self.options.with_os_api
+        elif self.settings.os == "baremetal":
```

Comment:
Oh! Didn't know this was a default setting in `conan.conf` 👍  https://github.com/conan-io/conan/pull/10067

Reply:
Yeah, it took a while to get that in. 🥳